src/Bicep.Cli/Services/TemplateEvaluation.cs

```diff
@@ -176,15 +176,19 @@ private static JToken EvaluateTemplate(JToken? templateJtoken, JToken? parameter
 
                 TemplateEngine.ValidateProcessedTemplate(template, "2020-10-01", deploymentScope);
 
-                return template.ToJToken();
+                var assertions = template.Asserts?.ToDictionary(p => p.Key, p => (bool)p.Value.Value) ?? new Dictionary<string, bool>();
+
+                return new Evaluation(template, assertions, null);
             }
             catch (Exception exception)
             {
-                throw new InvalidOperationException(
+                var error = new InvalidOperationException(
                     $"Evaluating template failed: {exception.Message}." +
```

Comment:
Returning the full template & parameter JSON we generate may be confusing to users. Have you thought about just returning `exception.Message` and making it a responsibility of the command to decide how to format it?